content/en/docs/getting-started/setup/create-cluster/eks.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Create EKS cluster
+linktitle: EKS
+description: How to create an EKS cluster?
+weight: 10
+categories: [getting started]
+keywords: [cluster]
+draft: true
```

Comment:
@daveconde Making the whole EKS docs a draft which means they won't be published for now.